### ISSUE TITLE
soci test projects for MSVC using cmake end up with wrong include path and linker error

### ISSUE THREAD

**Issue by hrabe** [SOCI/soci#177]
Each of the test project for Visual Studio end up with wrong defined _Additional Library Directories_. Instead getting the path to the required Boost Lib folder proper configured at cmake-ui I got this for all tests (not only ODBC but primary tested in may case):

```
E:\buildspace\builds\win32\DB2-8.2.3.0\lib\$(OutDir),E:\buildspace\builds\win32\DB2-8.2.3.0\lib
```
As example you will get this during compile:
```
Error	2	fatal error LNK1104: cannot open file 'libboost_date_time-vc80-mt-1_54.lib'	soci_firebird_test
```
Each test project gets configured with DB2 pathes instead of Boost. I did not find the place, were this went wrong.

**Comment by snikulov** [comment]
@hrabe @mloskot can we close this issue?

**Comment by mloskot** [comment]
Appveyor seems to not to have Boost enabled yet, so not sure if this has been actually fixed. Has it?

**Comment by snikulov** [comment]
@mloskot I think it can be fixed with 8fda2bb37a1da68878edced0fb75c94aec99e4fb.
Issue poorly informative about per-conditions and more to say - simply outdated :).

**Comment by mloskot** [comment]
OK, let's close.
